pkg/grpc/dapr/proto/common/v1/common.pb.go

```diff
@@ -12,12 +12,13 @@
 package common
 
 import (
+	reflect "reflect"
+	sync "sync"
+
```

Comment:
The `.pb.go` files under `pkg/grpc/dapr/proto/common/v1/` directory are automatically generated too.
Can we ignore all `.pb.go` files? If not, we can ignore the directory `pkg/grpc/dapr/proto/common/v1/`

Reply:
All .pb.go files have been ignored.